tests/unit/test_attribute_collection.py

```diff
@@ -0,0 +1,50 @@
+from unittest import TestCase
+
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+
+
+class TestAttributeCollection(TestCase):
+    def setUp(self):
+        auth = UsernamePasswordAuth("username", "password")
+        self.unify = Client(auth)
```

Comment:
For new code, use `tamr` instead of `unify`:

```suggestion
        self.tamr = Client(auth)
```